pkg/service/auth.go

```diff
@@ -38,6 +40,10 @@ const (
 	RESPONSE_MESSAGE_SERVICE_NOT_FOUND = "Service not found"
 
 	X_LOOKUP_KEY_NAME = "host"
+
+	envMaxHttpRequestBytes     = "MAX_REQUEST_BYTES"
```

Comment:
In that case, I'd go for extracting them to `helpers/envs.go` since having them in `main.go` would be just tied to the main package... and Golang doesn't allow cyclic dependencies. What do you think?

Reply:
We can inject the config in the AuthService, like we do with cache and others.